Add docs to show how to use custom images (#1323)
elastic/elastic-package: README.md

_Context: global_

> Use this command to spin up a Docker-based Elastic Stack consisting of Elasticsearch, Kibana, and the Package Registry. By default the latest released version of the stack is spun up but it is possible to specify a different version, including SNAPSHOT versions by appending --version <version>. In case, you need to run your own custom images (e.g. Elasticsearch, Kibana or elastic-agent), see [this document](./docs/howto/custom_images.md).

Reviewer: Nit.
```suggestion
Use this command to spin up a Docker-based Elastic Stack consisting of Elasticsearch, Kibana, and the Package Registry. By default the latest released version of the stack is spun up but it is possible to specify a different version, including SNAPSHOT versions by appending --version <version>.

You can run your own custom images for Elasticsearch, Kibana or elastic-agent, see [this document](./docs/howto/custom_images.md) for more information.
```

Use this command to spin up a Docker-based Elastic Stack consisting of Elasticsearch, Kibana, and the Package Registry. By default the latest released version of the stack is spun up but it is possible to specify a different version, including SNAPSHOT versions by appending --version <version>.

> You can run your own custom images for Elasticsearch, Kibana or elastic-agent, see [this document](./docs/howto/custom_images.md).

Reviewer: Nit. Actually if we use the uppercased names of the products we should probably use Elastic Agent.
```suggestion
You can run your own custom images for Elasticsearch, Kibana or Elastic Agent, see [this document](./docs/howto/custom_images.md).
```

Use this command to boot up the stack locally.

> By default the latest released version of the stack is spun up but it is possible to specify a different version, including SNAPSHOT versions by appending --version <version>. In case, you need to run your own custom images (e.g. Elasticsearch, Kibana or elastic-agent), see [this document](./docs/howto/custom_images.md).

Reviewer: Nit.
```suggestion
By default the latest released version of the stack is spun up but it is possible to specify a different version, including SNAPSHOT versions by appending --version <version>.

You can run your own custom images for Elasticsearch, Kibana or elastic-agent, see [this document](./docs/howto/custom_images.md) for more information.
```

By default the latest released version of the stack is spun up but it is possible to specify a different version, including SNAPSHOT versions by appending --version <version>.

> You can run your own custom images for Elasticsearch, Kibana or elastic-agent, see [this document](./docs/howto/custom_images.md).

Reviewer: Nit.
```suggestion
You can run your own custom images for Elasticsearch, Kibana or Elastic Agent, see [this document](./docs/howto/custom_images.md).
```
